sourcegraph/learn
# Add author profiles

## Type of change

### What should this PR do?
Resolves [DEVED-102](https://sourcegraph.atlassian.net/browse/DEVED-102) by adding author profiles.

Also removes the author index & related components, as we are not currently linking to the index from anywhere, and do not have current design plans for an index.

This is a minimal design for now that we can build out in the future.

### Why are we making this change?
Author profiles are a great way to allow authors to tell readers more about themselves; they also allow us to highlight authors' work across the site.

### What are the acceptance criteria? 
- Author profiles render correctly for each author slug.
- Links to author profiles work correctly from posts.
- Links to social media work correctly if provided.
- Design decisions make sense.
- Mobile views look correct.

### How should this PR be tested?
- Check out the branch, and check that the above criteria work and that everything looks okay on mobile.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
